projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -375,7 +375,7 @@ public function get_settings_response() {
 					$newsletter_categories   = maybe_unserialize( get_option( 'wpcom_newsletter_categories', array() ) );
 					$newsletter_category_ids = array_map(
 						function ( $newsletter_category ) {
-							return $newsletter_category['term_id'];
+							return $newsletter_category['term_id'] ?? $newsletter_category;
```

Comment:
I wonder if we should also do some check / casting on `$newsletter_category`, or can we always expect it to be in the right format?